Ginger/GingerCoreNET/WorkSpaceLib/WorkSpace.cs

```diff
@@ -376,6 +380,28 @@
                     return false;
                 }
 
+                //GetEncryptionKey(solution.Guid.ToString());
+                if (solution.FetchEncryptionKey())
+                {
+                    if(solution.ValidateKey())
+                    {
+                        IsEncryptionKeyValid = true;
+                    }
+                }
+                //Solution = solution;
+                if (!IsEncryptionKeyValid)
+                {
+                    Reporter.ToUser(eUserMsgKey.SolutionLoadError, "Encryption key validation failed or key is missing. Press Ok to enter the key manually.");
```

Comment:
Let's remove the pop up message and we will directly show the solution details page. May be the Encryption key Error indicator will be visible as soon as the page is open

Reply:
Removed pop up message.